src/backend/utils/misc/guc.c

```diff
@@ -2668,6 +2669,16 @@ static struct config_int ConfigureNamesInt[] =
 		NULL, NULL, NULL
 	},
 
+	{
+		{"sliding_window_step_factor", PGC_USERSET, QUERY_TUNING_OTHER,
+		 gettext_noop("Steps the default step size for a sliding window query as a factor of the window size."),
```

Comment:
~~Steps t~~The default step size for a sliding window query as a factor of the window size.